src/modules/simulator/simulator.h

```diff
@@ -315,12 +273,12 @@ class Simulator : public ModuleParams
 	static Simulator *_instance;
 
 	// simulated sensor instances
-	simulator::Report<simulator::RawAccelData>	_accel;
-	simulator::Report<simulator::RawMPUData>	_mpu;
-	simulator::Report<simulator::RawBaroData>	_baro;
-	simulator::Report<simulator::RawMagData>	_mag;
-	simulator::Report<simulator::RawGPSData>	_gps;
-	simulator::Report<simulator::RawAirspeedData> _airspeed;
+	simulator::Report<simulator::RawAccelData>	_accel{1};
```

Comment:
For next time it would be nice to separate refactor/cleanup work from functional changes and make a separate commit. That way it's easier to review.

Reply:
Yes you're right. Pragmatic laziness :smirk: 
https://github.com/PX4/Firmware/pull/11435

Reply:
:smile: